app/frontend/stylesheets/limber/screen.scss

```diff
@@ -280,6 +282,7 @@ nav.robots-list {
   .work-completion-button,
   .create-plate-button,
   .create-tube-button,
+  .create-tube_rack-button,
```

Comment:
Seems like some of this stuff could just be generic, like `.create-labware-button` or `params[:limber_labware_id]` in the above file. Let's not add refactoring to the scope of the story though!

Reply:
Agree, assuming the buttons don't have to have different text or some other difference.

Reply:
Plus the suggested action Add button now looks like 'Add <purpose> tube_rack' with an underscore which looks odd (works for tube and plate as they are single word classes).